admin/lib/tasks/importmap.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+namespace :solidus_admin do
+  namespace :importmap do
+    desc "Build Solidus Admin's Tailwind's css"
```

Comment:
Wrong description :slightly_smiling_face:

Reply:
LOL, now you know from where I copied it! 😜